README.md

```diff
@@ -1 +1,1205 @@
+[main]     Step 1/6 : FROM balenalib/fincm3-node:8
+[main]      ---> 392c3f6339f7
+[main]     Step 2/6 : WORKDIR /usr/src/app
+[main]     Using cache
+[main]      ---> 7d7f394b0c9c
+[main]     Step 3/6 : COPY package.json package-lock.json ./
+[main]      ---> d4e08e51860f
+[main]     Removing intermediate container e6d0ddf83498
+[main]     Step 4/6 : RUN npm install --ci --production     && npm cache clean --force     && rm -f /tmp/*
+[main]      ---> Running in 72b59e4ecd47
+[main]     added 50 packages from 37 contributors and audited 126 packages in 2.465s
+[main]     found 0 vulnerabilities
+[main]     npm
+[main]      WARN
+[main]      using --force
+[main]      I sure hope you know what you are doing.
+[main]
+[main]      ---> e2f81bc3ff91
+[main]     Removing intermediate container 72b59e4ecd47
+[main]     Step 5/6 : COPY . ./
+[main]      ---> 21b7d6b7f231
+[main]     Removing intermediate container 0658f893dadd
+[main]     Step 6/6 : CMD npm start
+[main]      ---> Running in b2dd4ccc8285
+[main]      ---> 83babbbb57b4
+[main]     Removing intermediate container b2dd4ccc8285
+[main]     Successfully built 83babbbb57b4
+[Info]     Uploading images
+[Success]  Successfully uploaded images
+[Success]  Release successfully created!
+[Info]     Release: 0576485d6a2049e066c424e1bab957a16e7cb396 (id: 1234567)
+[Info]     ┌─────────┬────────────┬────────────┐
+[Info]     │ Service │ Image Size │ Build Time │
+[Info]     ├─────────┼────────────┼────────────┤
+[Info]     │ main    │ 208.39 MB  │ 7 seconds  │
+[Info]     └─────────┴────────────┴────────────┘
+[Info]     Build finished in 15 seconds
+			    \
+			     \
+			      \\
+			       \\
+			        >\/7
+			    _.-(6'  \
+			   (=___._/` \
+			        )  \ |
+			       /   / |
+			      /    > /
+			     j    < _\
+			 _.-' :      ``.
+			 \ r=._\        `.
+			<`\\_  \         .`-.
+			 \ r-7  `-. ._  ' .  `\
+			  \`,      `-.`7  7)   )
+			   \/         \|  \'  / `-._
+			              ||    .'
+			               \\  (
+			                >\  >
+			            ,.-' >.'
+			           <.'_.''
+			             <'
+
+To git.balena-cloud.com:heds/cliapp.git
+ * [new branch]      initial-pr -> master
+ ```
+
+This will store the code from the local repository into the private copy
+stored by balena.
+
+Just like `balena push`, once the code has been built it will be downloaded
+by the device and then run.
+
+**Important Note:** There is absolutely no guarantee that the private repository
+will be available for pulling code from, and as such it should never be used
+as an application's source repository origin. It is intended to be a temporary
+repository used only for pushing to the builders.
+
+## 4. SSHing into a Device
+
+Once a device has been provisioned, it can be accessed by SSHing into it via
+the balenaClound VPN. To do this, you'll need the UUID of the device you want to
+SSH into (remember you can see all your devices by running `balena devices`):
```

Comment:
so there's a few times you mention uuid, it might be worth adding at the first instance of a uuid being used that this is the short uuid, and the long uuid is also usable.

Reply:
Done.